### PR TITLE
🚮[Amp story] Delete desktop 3 panel and viewport warning code.

### LINKED ISSUE #35623: [Amp story] Remove desktop 3 panel UI and associated code.
Remove all code related to 3 panel desktop UI including viewport warnings.

### PR DESCRIPTION
**Pull request by processprocess**
Deletes desktop three panel code.
Deletes viewport warning code.

Fixes #35623

### CODE REVIEW

**Review by mszylkowski — COMMENTED** [review]
We should also update the [`ampdoc.css`](https://github.com/ampproject/amphtml/blob/main/css/ampdoc.css#L272) rules so the old bg color doesn't ever flash

File: extensions/amp-story/1.0/amp-story-page.js [hunk 1]
```diff
@@ -542,19 +542,6 @@ export class AmpStoryPage extends AMP.BaseElement {
...
-      // The rewinding is delayed on desktop so that it happens at a lower
-      // opacity instead of immediately jumping to the first frame. See #17985.
-      this.pauseAllMedia_(false /** rewindToBeginning */);
-      this.timer_.delay(() => {
-        this.rewindAllMedia_();
-      }, REWIND_TIMEOUT_MS);
-    } else {
-      this.pauseAllMedia_(true /** rewindToBeginning */);
-    }
-
```
Comment (mszylkowski) [inline extensions/amp-story/1.0/amp-story-page.js:557]: Shouldn't this `this.pauseAllMedia_` branch stay? If removed, I'd think the videos wouldn't rewind or pause.

Comment (processprocess) [reply extensions/amp-story/1.0/amp-story-page.js:557]: YES! Thank you good catch.

**Comment by processprocess** [comment]
> We should also update the [`ampdoc.css`](https://github.com/ampproject/amphtml/blob/main/css/ampdoc.css#L272) rules so the old bg color doesn't ever flash

Yes! Good call.

File: extensions/amp-story/1.0/amp-story-page.js [hunk 2]
```diff
@@ -1179,6 +1157,7 @@ export class AmpStoryPage extends AMP.BaseElement {
           promises.push(mediaPool.play(mediaEl));
         }
 
+        this.pauseAllMedia_(true /** rewindToBeginning */);
```
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-page.js:1160]: This should go L544?

**Review by gmajoulet — APPROVED** [review]

**Comment by calebcordry** [comment]
could you please also clean up rules in `amp-story-auto-ads-ad-badge.css` and `amp-story-auto-ads-progress-bar.css`? the rules using `[desktop-panels]` can be deleted.

**Review by calebcordry — APPROVED** [review]

**Comment by processprocess** [comment]
cc @dmanek for OWNERS on css/ampdoc.css

**Review by dmanek — APPROVED** [review]
`css/ampdoc.css` changes LGTM

**Review by mszylkowski — APPROVED** [review]
🚢

**Pull request merged**